CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# CorisxTH Contributing Guidelines #
+
+## Tell other devs what you are working on! ##
+
+When you start working on a fix/enhancement for an open issue please always
+post a comment in this issue's discussion to tell other devs that you have
+started working on it so that they won't waste any of their free time by
+working on their own fix/enhancement for it.
+
+If there isn't an open issue discussion for a bug fix/enhancement you want
+to work on please check that there isn't a closed discussion for it because
+a previous developer of it didn't finish their work which you could finish.
+
+And if there is no existing issue discussion for what you want to work on
+please open a new issue for it and tell other devs here that your going to/
+have started working on it.
+
```

Comment:
Suggest to ask or discuss in the Matrix channel as well in case of not being sure the new feature is wanted?

Reply:
That's the point of submitting a Feature Request  as below

Reply:
But you're right, just adding an enhancement PR without reason could be peculiar. Not that we could stop people doing it. Perhaps instead we should suggest making a Feature Request first.

Reply:
I think we do want to stop people from doing it, drive by feature PRs for things we haven't discussed tend to waste everyone's time.